monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -16,16 +34,34 @@ class SCDTestIssue(ImplicitDict):
     details: str
     '''Human-readable description of the issue'''
 
-    def __init__(self, **kwargs):
-        super(SCDTestIssue, self).__init__(**kwargs)
-        if 'timestamp' not in kwargs:
-            self.timestamp = datetime.datetime.utcnow().isoformat()
+    queries: List[fetch.Query]
+    """Description of HTTP requests relevant to this issue"""
 
 class RequiredResult(ImplicitDict):
     ''' A class to evaluate results / response to an injection of test flight data (TestFlightRequest) '''
-    expected_result: str
-    incorrect_result_details: Optional[SCDTestIssue]
+    acceptable_results: List[str] # Holds information about the "result" response provided by the test interface of the USS under test. Per the SCD Testing API this should be one of "Planned", "Rejected", "ConflictWithFlight" or "Failed"
```

Comment:
Let's try to be consistent with style within a single file at a minimum.  Above classes use triple-single-quote pseudo-docstrings on the line below the declaration whereas this class uses comments after the line.

Reply:
Thanks yes, good point, I have now made it all single quotes.

Reply:
As of 39cbc58, this comment has not been addressed.

Reply:
Thanks for this noted, I misunderstood your comment, I have now made the changes.